src/core/plugins/react-17/root-injects.js

```diff
@@ -0,0 +1,5 @@
+export const render = (getSystem, getStore, getComponent, getComponents) => (domNode) => {
+    const App = getComponent(getSystem, getStore, getComponents)("App", "root")
+    const ReactDOM = require("react-dom")
+    ReactDOM.render(<App/>, domNode)
+}
```

Comment:
You have to export the plugin:

```suggestion
}

export default React17Plugin
```